docs/codeSnippets.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Removes the initial prompt characters "$ " from the beginning of each line in a multiline string, if present.
+ *
+ * @param {string} text - The input string, potentially containing multiple lines with prompts at the beginning.
+ * @returns {string} - The modified string with initial prompts removed from each line, or the original lines if no prompt is present.
+ */
+function removePrompt(text) {
+    return text.split('\n').map(line => {
+        if (line.startsWith("$ ")) {
+            return line.slice(2);
+        }
+        return line;
+    }).join('\n');
+}
+
```

Comment:
Shouldn't it just remove the leading `$`?

Reply:
No because we can have multiline snippets, e.g.
```shell
$ echo abc
$ echo def
```

Reply:
Okay, do we have any snippets like this at the moment?

Reply:
Yes, e.g. [here](https://github.com/foundry-rs/starknet-foundry/blob/franciszekjob/2479-expandable-shell-snippets/docs/src/getting-started/first-steps.md).

![image](https://github.com/user-attachments/assets/fdbbca43-a517-4b0d-b334-033c7efe9097)